Formula/abcmidi.rb

```diff
@@ -21,7 +21,7 @@ def install
     # configure creates a "Makefile" file. A "makefile" file already exist in
     # the tarball. On case-sensitive file-systems, the "makefile" file won't
     # be overridden and will be chosen over the "Makefile" file.
-    rm "makefile"
+    rm_f "makefile"
```

Comment:
Can we try removing this line and comment altogether?  In the source archive it looks like they renamed `makefile` to `makefile.w32`, which would mean the entire workaround is unneeded.

Reply:
That sounds about right: :)

```
$ ls -1 makefile*
Makefile
Makefile.in
makefile.w32
```